### PR TITLE
SAK-27778 - Chat - please use new date picker for consistency

### PR DESCRIPTION
**Pull request by mpellicer**
Chat - please use new date picker for consistency

### CODE REVIEW

File: chat/chat-tool/tool/src/webapp/jsp/editRoom.jsp [hunk 1]
```diff
@@ -2,7 +2,31 @@
...
+             localDatePicker({
+                input: '#editRoomForm\\:startDate',
+                useTime: 0,
+                parseFormat: 'YYYY-MM-DD',
+                allowEmptyDate: true,
+                val: '<h:outputText value="#{ChatTool.currentChannelEdit.startDate}"><f:convertDateTime pattern="yyyy-MM-dd"/></h:outputText>',
+                ashidden: { iso8601: 'chatStartDateISO8601' }
+            });
+          });
+          $(document).ready(function() {
```
Comment (ottenhoff) [inline chat/chat-tool/tool/src/webapp/jsp/editRoom.jsp]: document.ready should only be done once per file

**Comment by mpellicer** [comment]
Thanks @ottenhoff, PR updated!

**Comment by mpellicer** [comment]
@ottenhoff It fails because the format is inside the properties:

msgs.custom_date_entry_format

I'll suggest to use the ISO pattern format in the client and the server ignoring the display format.

**Comment by mpellicer** [comment]
PR updated using always the ISO format.

**Comment by ottenhoff** [comment]
This works well for me in EN and SV!

**Comment by jonespm** [comment]
Yeah, better solution. It seems better if "generic looking" utilitiy classes like this should be bumped up into kernel-util or portal-util rather than duplicated since you seem to be using it across multiple PR's. Just easier to fix a bug in one place than multiple, and we'll forget if it's not done now. (Though if you want to clean all that up on a separate ticket once these are in that's okay too)

**Comment by mpellicer** [comment]
@jonespm Yeah i thought the same, but better if we get the PRs into Master first!

After all of them merged, i'll move the DateUtil class to kernel-util.

Miguel

**Pull request merged**